src/test/utils/MockOwnedAdapter.sol

```diff
@@ -42,9 +42,13 @@ contract MockOwnableAdapter is OwnableAdapter, Trust {
         uint256 uDecimals = underlying.decimals();
 
         underlying.transferFrom(msg.sender, address(this), uBal);
-        amountOut = uDecimals < tDecimals ? 
-            uBal * 1e18 / scale * (tDecimals - uDecimals) ** 10 :
-            uBal * 1e18 / scale / (uDecimals - tDecimals) ** 10;
+        if (tDecimals == uDecimals) {
```

Comment:
@fedealconada I had to add this case to the mock adapter since otherwise I ran into a division by 0 error. I know you've spent some time with the mocks, so just wanted to give u a ping to see if that seems right to you? :)

Reply:
yeah, looks alright to me